src/styles/index.ts

```diff
@@ -4156,6 +4156,86 @@ const styles = (theme: ThemeColors) =>
             borderColor: theme.icon,
         },
 
+        interactiveStepHeaderContainer: {
+            flex: 1,
+            alignSelf: 'center',
+            flexDirection: 'row',
+            paddingBottom: 24,
+        },
+
+        interactiveStepHeaderStepContainer: {
+            flexDirection: 'row',
+            alignItems: 'center',
+        },
+
+        interactiveStepHeaderStepButton: {
+            width: 40,
+            height: 40,
+            borderWidth: 1,
+            borderRadius: 20,
+            borderColor: colors.green400,
+            justifyContent: 'center',
+            alignItems: 'center',
+            color: colors.white,
+        },
+
+        interactiveStepHeaderLockedStepButton: {
+            borderColor: colors.productDark400,
+        },
+
+        interactiveStepHeaderStepText: {
+            fontSize: variables.fontSizeLabel,
+            fontFamily: FontUtils.fontFamily.platform.EXP_NEUE,
+            fontWeight: FontUtils.fontWeight.bold,
+        },
+
+        interactiveStepHeaderCompletedStepButton: {
+            backgroundColor: colors.green400,
+        },
+
+        interactiveStepHeaderStepLine: {
+            height: 1,
+            flexGrow: 1,
+            backgroundColor: colors.green400,
+        },
+
+        interactiveStepHeaderLockedStepLine: {
+            backgroundColor: colors.productDark400,
+        },
+        confirmBankInfoCard: {
+            backgroundColor: colors.green800,
+            borderRadius: variables.componentBorderRadiusCard,
+            marginBottom: 20,
+            marginHorizontal: 16,
+            padding: 20,
+            width: 'auto',
+            textAlign: 'left',
+        },
```

Comment:
Can you keep the same spacing in this file as in the rest of it? thanks!
```suggestion
        },

```